### ISSUE TITLE
7.2.1 add option to remove guild bank from item count

### ISSUE THREAD

**Issue by ceylina** [Jaliborc/Bagnon#542]
Can you add an option to remove guild bank items from the item count?

**Comment by Raxol** [comment]
I would like to see this option aswell. It's rather confusing to see the guild bank count. It also gives a wrong total count if some of the alts are in the same guild for example.

**Comment by Jaliborc** [comment]
Ok, will do.

**Comment by sidelsky18** [comment]
Thank you!

**Comment by SirThrocken** [comment]
Thank you!  Too many numbers now, especially with all my toons in the same guild!

**Comment by Kumomo** [comment]
Not sure if this would belong in its own suggestion post or if it's okay here, but would it be possible to make the Guild count listed by itself (as if it was a character)? I'm not sure if this would also help with the odd item total counts when alts don't have the guild bank count renewed or if it would be more difficult to implement.

**Comment by voidzone** [comment]
Yes, please give us the option to disable guild bank totals or make it calculate the total based off the guild name rather than for each character you have in the guild.

**Comment by LordGrizzard** [comment]
Yes please, I don't care that there are 1786 Silk Cloth in my Guild Bank, i just want to know how much I have in my Tailot's Bank.

**Comment by anaander** [comment]
Not a log of progress seems to have been made on this but if you want to remove this feature, you can edit Bagnon\external\Wildpants\libs\LibItemCache-1.1\Core.lua.

change line 256:
`return equip, bags - equip, total - bags, vault or 0, guild or 0`
to:
`return equip, bags - equip, total - bags, vault or 0`

**Comment by voidzone** [comment]
> 1 string of code and it's been a year already, please add this :<

It's been an option for a while now.  Open the top level of the config and uncheck the "Include Guild Banks" checkbox.

https://imgur.com/a/LqBh7VE

**Comment by LordGrizzard** [comment]
The post is almost 1 year old xD
